### ISSUE TITLE
ScreenReader(NVDA + chrome) does not announce header cell of the table

### ISSUE THREAD

**Issue by srjonnal** [angular-ui/ui-grid#7082]
While focus is on the header cell of the table, screen reader announces as Column menu. It should be announced as header cell. 

![image](https://user-images.githubusercontent.com/54173649/79122560-9a547400-7db5-11ea-82ac-4de637c3bfd4.png)

**Comment by kanyalka** [comment]
Adding a comment to reactivate this issue, as it has not been fixed.

**Comment by srjonnal** [comment]
Reactiving since this issue hasn't been fixed yet.
